llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -1306,6 +1389,8 @@ Expected<uint64_t> MCCASReader::materializeSection(cas::ObjectRef ID) {
     OS.write_zeros(1);
     return *Size + 1;
   }
+  if (auto F = DebugLineRef::Cast(*Node))
+    return F->materialize(OS);
```

Comment:
@rastogishubham have you actually _seen_ any DWARF debug info sections with multiple fragments? Nowhere in the code I see how the original fragments are materialized in their original state, which makes me believe we're not testing the multiple-fragment case.

Reply:
If you step through the code for Debug Line, you will see that there are at least 10 - 12 fragments for the debug line section, in a small contrived test like 

`int bar() {
   return 1;
   }
int foo() {
 return 2;
int baz() {
  return 3;
`
So yes, I have seen multiple fragments, and I think we shouldn't assume that there will always be one unless we find solid proof in the code that there is only one